README.md

```diff
@@ -170,6 +170,34 @@ To do this, add the following to your `.vscode/settings.json` file:
 },
 ```
 
+## Zed Setup
+
+Because this repository uses a custom sysroot with custom libraries and binaries, you need to set up your environment accordingly.
```

Comment:
What about having an “IDE configuration” section where we put this sentence, then subsections for VSCode and Zed?

Reply:
Done.

Reply:
I'd still move this sentence before the VSCode subsection, to avoid repeating the same thing twice. (Not blocking, though)